### ISSUE TITLE
Withdraw all funds not working correctly - removing open orders significantly delayed, available funds showing negative balance and fee reserve not being withdrawn

### ISSUE THREAD

**Issue by matt-bullock** [AugurProject/augur#8279]
After withdrawing all funds. Note the negative balance in the top bar, open orders still showing and fee reserve still there.

![image.png](https://images.zenhubusercontent.com/5c07af47035895308911c4c5/c7bd39e9-2c02-40ef-b804-4ada4b445bc4)

After quite a few minutes delay the negative balance was removed as well as the open orders, but fee reserve is still showing

![image.png](https://images.zenhubusercontent.com/5c07af47035895308911c4c5/3334a32f-4fe4-40d8-bba3-72d3da67b97e)

**Comment by bthaile** [comment]
It takes a second for open orders to be cancelled, it's done on zeroX side and the UI is informed, I saw a negative balance for less than one second. Then my open orders were removed and I saw the below.

![image](https://user-images.githubusercontent.com/3970376/86469610-5b242800-bcff-11ea-8758-f34c0165f1bc.png)


ETH didn't get withdrawn. Reason is the withdraw all funds gas estimation fails, so a simple transfer is used which only transfers DAI.
